### DISCUSSION TITLE
Add “save kmz timeseries.py” script in “Time series  analysis with HyP3 and Mintpy” jupiter notebook

### DISCUSSION THREAD

**Discussion by Pierparide** [insarlab/MintPy#944] [Q&A]
I've succefully completed the two jupiter notebook named "Prepare HyP3 Insar Stack for Mintpy" and "Mintpy Time Series From Prepared Data stack" but my intention is add save kmz timeseries script in jupiter notebook to convert displacement time series. Anyone can help me?

**Comment by yunjunz** [comment]
The UTM coordinate system, as used in HyP3, is currently not supported in `save_kmz.py` and `save_kmz_timeseries.py` (https://github.com/insarlab/MintPy/issues/686).

**Reply by Pierparide** [reply]
There is another way to obtain same kmz interactive displacement from raster?
